code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -775,3 +755,35 @@
 	var/obj/item/pushbroom/cyborg/BR = locate() in R.model.modules
 	if (BR)
 		R.model.remove_module(BR, TRUE)
+
+///This isn't an upgrade or part of the same path, but I'm gonna just stick it here because it's a tool used on cyborgs.
+/obj/item/borg_restart_board
+	name = "cyborg emergency reboot module"
+	desc = "Used to force a reboot of a disabled-but-repaired cyborg, bringing it back online."
```

Comment:
can you put something in here that clarifies its reusable unlike the other boards